modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1700,6 +1700,14 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
   protected abstract getAddressInfoFromExplorer(address: string, apiKey?: string): Bluebird<AddressInfo>;
   protected abstract getUnspentInfoFromExplorer(address: string, apiKey?: string): Bluebird<UnspentInfo[]>;
 
+  getUnspentFromWrongChain(faultyTxId: string): any {
```

Comment:
Name here seems a bit strange - maybe something like `getTxInfoFromExplorer` would be a bit more descriptive? As it is, this doesn't have much to do with unspents or wrong chains.

Reply:
Yeah that sounds good. The tests used to the "Wrong Chain" as the descriptor, which led me to name it that. Going to change now.

Reply:
Change pushed @tylerlevine